README.md

```diff
@@ -2,6 +2,39 @@
 
 [![Scala.js](https://www.scala-js.org/assets/badges/scalajs-0.6.6.svg)](https://www.scala-js.org/)
 
+## Installing
+
+First clone this repository locally and then publish locally
+```
+> ++SCALA_VERSION
```

Comment:
Actually, this should be `++2.10.6`, because it will be used by the *build*, not the project.

That also means we should cross-test this with `2.10.6` ^^